package.json

```diff
@@ -18,7 +18,7 @@
     "test": "npm run build:production -- --display=none && bundlesize",
     "test:visualtest": "npm run build:static && start-server-and-test storybook http-get://localhost:9001 backstopjs",
     "test:visualapprove": "backstop approve --config=tests/backstop/config.js",
-    "test:analyze": "npm run build:production -- --env.analyze=true --json --progress --profile > /tmp/stats.json && cp ./dist/assets/js/*.js /tmp && webpack-bundle-analyzer /tmp/stats.json",
+    "test:analyze": "npm run remove-dist && mkdir -p ./dist/assets/js && webpack --mode=production --env.analyze=true --json --profile > ./dist/assets/js/stats.json && webpack-bundle-analyzer ./dist/assets/js/stats.json",
```

Comment:
This is beyond the scope of this issue; if there is a case for making these changes, let's open a new issue to discuss it 🙂